script/_dhcpv6_pd_ref

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+customise_dhcpcd_conf()
+{
+    sudo tee /etc/dhcpcd.conf >/dev/null <<EOF
+# A sample configuration for dhcpcd.
+# See dhcpcd.conf(5) for details.
+
+# Allow users of this group to interact with dhcpcd via the control socket.
+#controlgroup wheel
+
+# Inform the DHCP server of our hostname for DDNS.
+#hostname
+
+# Use the hardware address of the interface for the Client ID.
+#clientid
+# or
+# Use the same DUID + IAID as set in DHCPv6 for DHCPv4 ClientID as per RFC4361.
+# Some non-RFC compliant DHCP servers do not reply with this set.
+# In this case, comment out duid and enable clientid above.
+#duid
+
+# Rapid commit support.
+# Safe to enable by default because it requires the equivalent option set
+# on the server to actually work.
+option rapid_commit
+
+# A ServerID is required by RFC2131.
+#require dhcp_server_identifier
+
+noipv6rs 
+# disable routing solicitation
+# denyinterfaces eth1  # Don't touch other_interfaces at all
+interface eth0
+  ipv6rs
+  iaid 1
+  ia_pd 2/::/64 -
+
+interface wlan0
+  ipv6rs
+  iaid 2
+  ia_pd 3/::/64 -
+EOF
+}
+
+dhcpv6_pd_ref_stop()
+{
+    if [[ -f "/etc/dhcpcd.conf.orig" ]]; then
+        sudo mv /etc/dhcpcd.conf.orig /etc/dhcpcd.conf
+    fi
+    # sudo systemctl restart dhcpcd
+    sudo rm -f /etc/dhcpcd.enter-hook /etc/dhcpcd.exit-hook
+
+}
+
+dhcpv6_pd_ref_start()
+{
```

Comment:
The `install/uninstall` actions should only happen when the env variable is on. Like:
https://github.com/openthread/ot-br-posix/blob/3e9be206df0860a7a63b7ad82181f95f8ca2fbc7/script/_firewall#L38